{"trial_id": "0099", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "num_layers", "default": null, "conditions": [], "min_value": 0, "max_value": 2, "step": 1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "units_0", "default": null, "conditions": [], "min_value": 1, "max_value": 320, "step": 1, "sampling": "linear"}}, {"class_name": "Choice", "config": {"name": "dropout", "default": 0.0, "conditions": [], "values": [0.0, 0.1, 0.2, 0.3, 0.4, 0.5], "ordered": true}}, {"class_name": "Int", "config": {"name": "units_1", "default": null, "conditions": [], "min_value": 1, "max_value": 320, "step": 1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "units_2", "default": null, "conditions": [], "min_value": 1, "max_value": 320, "step": 1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "epochs", "default": null, "conditions": [], "min_value": 1, "max_value": 100, "step": 1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "batch_size", "default": null, "conditions": [], "min_value": 1, "max_value": 256, "step": 1, "sampling": "linear"}}], "values": {"num_layers": 1, "units_0": 20, "dropout": 0.5, "units_1": 317, "units_2": 76, "epochs": 49, "batch_size": 26, "tuner/epochs": 4, "tuner/initial_epoch": 2, "tuner/bracket": 4, "tuner/round": 1, "tuner/trial_id": "0075"}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"/scratch/nas/3/agirones/tfm/myenv/lib/python3.10/site-packages/keras_tuner/engine/base_tuner.py\", line 270, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"/scratch/nas/3/agirones/tfm/myenv/lib/python3.10/site-packages/keras_tuner/engine/base_tuner.py\", line 235, in _run_and_update_trial\n    results = self.run_trial(trial, *fit_args, **fit_kwargs)\n  File \"/scratch/nas/3/agirones/tfm/myenv/lib/python3.10/site-packages/keras_tuner/tuners/hyperband.py\", line 425, in run_trial\n    return super().run_trial(trial, *fit_args, **fit_kwargs)\n  File \"/scratch/nas/3/agirones/tfm/myenv/lib/python3.10/site-packages/keras_tuner/engine/tuner.py\", line 287, in run_trial\n    obj_value = self._build_and_fit_model(trial, *args, **copied_kwargs)\n  File \"/scratch/nas/3/agirones/tfm/myenv/lib/python3.10/site-packages/keras_tuner/engine/tuner.py\", line 213, in _build_and_fit_model\n    model = self._try_build(hp)\n  File \"/scratch/nas/3/agirones/tfm/myenv/lib/python3.10/site-packages/keras_tuner/engine/tuner.py\", line 155, in _try_build\n    model = self._build_hypermodel(hp)\n  File \"/scratch/nas/3/agirones/tfm/myenv/lib/python3.10/site-packages/keras_tuner/tuners/hyperband.py\", line 432, in _build_hypermodel\n    model.load_weights(self._get_checkpoint_fname(trial_id))\n  File \"/scratch/nas/3/agirones/tfm/myenv/lib/python3.10/site-packages/keras/src/utils/traceback_utils.py\", line 70, in error_handler\n    raise e.with_traceback(filtered_tb) from None\n  File \"/scratch/nas/3/agirones/tfm/myenv/lib/python3.10/site-packages/tensorflow/python/training/py_checkpoint_reader.py\", line 31, in error_translator\n    raise errors_impl.NotFoundError(None, None, error_message)\ntensorflow.python.framework.errors_impl.NotFoundError: Unsuccessful TensorSliceReader constructor: Failed to find any matching files for hpo/tfm/trial_0075/checkpoint\n"}